WooCommerce/Classes/ViewRelated/Products/Variations/ProductVariationsViewController.swift

```diff
@@ -287,10 +291,26 @@ private extension ProductVariationsViewController {
         addVariationButton.translatesAutoresizingMaskIntoConstraints = false
         addVariationButton.setTitle(Localization.addNewVariation, for: .normal)
         addVariationButton.addTarget(self, action: #selector(addButtonTapped), for: .touchUpInside)
-        addVariationButton.applySecondaryButtonStyle()
+        addVariationButton.applyPrimaryButtonStyle()
 
         buttonContainer.addSubview(addVariationButton)
-        buttonContainer.pinSubviewToSafeArea(addVariationButton, insets: .init(top: 16, left: 16, bottom: 16, right: 16))
+        buttonContainer.pinSubviewToSafeArea(addVariationButton, insets: .init(top: 16, left: 16, bottom: 8, right: 16))
```

Comment:
I that these insets are now duplicated. I think it would be good to abstract it in our `Settings` enum. What do you think?

Reply:
Actually, the insets are subtly different between the two buttons - if you look closely you can see that the Add Variation button's bottom space is 8 points, and the Edit Attribute button's top space is 8 points. I did this to make sure the space between the two buttons is 16 points.

But I get the point that there's code duplication and that smells - so I committed a new change to make top button configuration logic reusable - please check that out and let me know what you think!